charts/edge/templates/statefulset.yaml

```diff
@@ -64,6 +65,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
+          - name: POD_INDEX_LABEL
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.labels['apps.kubernetes.io/pod-index']
```

Comment:
TIL! Fancy

Is there any use or is it for future proofing?

Reply:
It gets used (if available) in the payload constructed in the configmap.
https://github.com/mezmo/helm-charts/pull/8/files#diff-510bc4db2cca87e7ca52f27796d6d663066eea557e6ca6f2012e97172d8b833aR17-R23

The alternative would be to inject pod name and parse it out in code. I didn't go that far because
- I expect to advise scaling up (not out) per instance, most times
- Trying to minimize coupling in vector to the heartbeat payload
- Hoping people are running fairly current k8s
- Heartbeat collisions are almost meaningless on this dimension -- we only lose the perception that there are two nodes (not one), and there is no expected change in UX